bootcamp/tutorials/integration/openai_agents_milvus.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<p>"upgrade of our previous experimentation&nbsp;" -&gt; "upgrade of its previous experimentation&nbsp;"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>